src/modules/task/task.controller.ts

```diff
@@ -24,6 +25,12 @@ const TaskController = {
     const { id } = validation(UpdateTaskParamsDto, req.params);
     const result = TaskService.update(id, dto);
 
+    res.json(result);
+  },
+  getById(req: Request, res: Response) {
```

Comment:
```suggestion
  },

  getById(req: Request, res: Response) {
```
Опять все сплошняком лепишь